xavier/poc/xavier-local.py

```diff
@@ -0,0 +1,76 @@
+import speech_recognition
+import snowboydecoder
+import sys
+import signal
+# import pyttsx3
+import os
+
+interrupted = False
+
+def signal_handler(signal, frame):
+    global interrupted
+    interrupted = True
+
+
+def interrupt_callback():
+    global interrupted
+    return interrupted
+
+# Define what to say
+def respond(audio) :
+    print(audio)
+    for line in audio.splitlines() :
+        os.system('/home/jay/git/mimic1/mimic -t "' + audio + '"')
```

Comment:
Don't hardcode a path specific to your machine.

Reply:
Oops. Fixed